src/1060_len_310.c

```diff
@@ -6,9 +6,9 @@ u32 func_80025D74(void* arg0, const unsigned char* str, s32 count);
 void func_80025C60(void) {
     OSPiHandle** handle = &carthandle;
 
-    osEPiWriteIo(*handle, 0xB3FF0014, 0);
-    osEPiWriteIo(*handle, 0xB3FF0004, 0);
-    osEPiWriteIo(*handle, 0xB3FF0000, ASCII_TO_U32('I', 'S', '6', '4'));
+    osEPiWriteIo(carthandle, 0xB3FF0014, 0);
```

Comment:
Because the `handle` temp is still needed, I think it's weird to leave it there unused. I'd rather just use the temp until we figure this out